### PR TITLE
OCLOMRS-395: Remove current implementation of “remove mapping” and "edit mapping"

### PR DESCRIPTION
**Pull request by samdiano**
# JIRA TICKET NAME:
[Remove current implementation of “remove mapping” and "edit mapping"](https://issues.openmrs.org/browse/OCLOMRS-395)

# Summary:
the "remove mapping" and "edit mapping" feature was implemented on the wrong interface and re-implemented on the correct interface, It should be removed.

### CODE REVIEW

File: src/components/dictionaryConcepts/components/MappingModal.jsx [hunk 1]
```diff
@@ -45,17 +45,6 @@ export class MappingModal extends Component {
     });
   };
 
-  EditMapping = () => {
```
Comment (dessHub) [inline src/components/dictionaryConcepts/components/MappingModal.jsx]: Why delete this? Is related to the ticket?

File: src/components/dictionaryConcepts/components/MappingModal.jsx [hunk 2]
```diff
@@ -101,8 +91,6 @@ export class MappingModal extends Component {
             )}
           </ModalBody>
           <ModalFooter>
-            <Button color="primary" id="mappingSubmit" onClick={to_concept_name !== '' ? this.EditMapping : null}>Save</Button>
```
Comment (Tittoh) [inline src/components/dictionaryConcepts/components/MappingModal.jsx]: Is this related to this ticket?

File: src/components/dictionaryConcepts/components/MappingModal.jsx (hunk 1 above)
Comment (samdiano) [reply src/components/dictionaryConcepts/components/MappingModal.jsx]: Yes. It is related to the ticket

File: src/components/dictionaryConcepts/components/MappingModal.jsx (hunk 2 above)
Comment (samdiano) [reply src/components/dictionaryConcepts/components/MappingModal.jsx]: Yes. The Save button has to be removed because the Edit functionality is being removed too

**Comment by dkayiwa** [comment]
@samdiano why is this still open for a full week?

**Comment by samdiano** [comment]
> @samdiano why is this still open for a full week?

@dkayiwa I was waiting for the ticket that made the implementation of edit mapping on the right interface before I communicate for it to be merged. The ticket was just merged today

**Comment by dkayiwa** [comment]
Did you look into the coverage decrease?

**Comment by samdiano** [comment]
> Did you look into the coverage decrease?

Yes I looked into the coverage decrease, there are no files related to my ticket that needs to be covered

**Comment by samdiano** [comment]
> Did you look at the travis failure?

Yes, I looked at it. I have rebased against master and that fixed it

File: src/components/dictionaryConcepts/components/ExternalMapping.jsx [hunk 3]
```diff
@@ -28,6 +28,7 @@ class ExternalMapping extends Component {
                 onChange={this.handleChange}
                 id="map_type"
                 required
+                disabled
```
Comment (dkayiwa) [inline src/components/dictionaryConcepts/components/ExternalMapping.jsx]: What is disabled doing here?

File: src/components/dictionaryConcepts/components/ViewMappingsModal.jsx [hunk 4]
```diff
@@ -67,19 +64,14 @@ export class ViewMappingsModal extends Component {
                         editMapping,
                         concepts,
                         mappings,
-                        buttonName: 'Edit',
+                        buttonName: 'View',
```
Comment (dkayiwa) [inline src/components/dictionaryConcepts/components/ViewMappingsModal.jsx]: Is this change part of the ticket?

File: src/components/dictionaryConcepts/components/ExternalMapping.jsx (hunk 3 above)
Comment (samdiano) [reply src/components/dictionaryConcepts/components/ExternalMapping.jsx]: disabled was put there to make the select option uneditable since the edit was removed already.

File: src/components/dictionaryConcepts/components/ViewMappingsModal.jsx (hunk 4 above)
Comment (samdiano) [reply src/components/dictionaryConcepts/components/ViewMappingsModal.jsx]: Yes, it was part of this ticket, I replaced the edit button with view as the edit function was removed.

After getting the response from Darius concerning this ticket, I will have to increase the scope of what the ticket handles. I need to remove the View mapping button that leads to the modal, I will need to remove the entire modal too.

Can I do that within this ticket also?

Comment (dkayiwa) [reply src/components/dictionaryConcepts/components/ViewMappingsModal.jsx]: Remove as much as required by the ticket.

File: src/components/dictionaryConcepts/components/ExternalMapping.jsx (hunk 3 above)
Comment (dkayiwa) [reply src/components/dictionaryConcepts/components/ExternalMapping.jsx]: Was the goal to just disable or completely remove?

Comment (samdiano) [reply src/components/dictionaryConcepts/components/ExternalMapping.jsx]: The goal was to remove completely

File: src/components/dictionaryConcepts/components/ViewMappingsModal.jsx (hunk 4 above)
Comment (samdiano) [reply src/components/dictionaryConcepts/components/ViewMappingsModal.jsx]: I have updated my changes

File: src/tests/dictionaryConcepts/container/CreateConcept.test.jsx [hunk 5]
```diff
@@ -157,6 +157,8 @@ describe('Test suite for dictionary concepts components', () => {
   it('should handle form completion and submission with correct data', () => {
     const conceptName = { target: { name: 'name', value: 'test concept' } };
     wrapper.find('#concept-name').simulate('change', conceptName);
+    const conceptClass = { target: { name: 'concept_class', value: 'Procedure' } };
```
Comment (dkayiwa) [inline src/tests/dictionaryConcepts/container/CreateConcept.test.jsx]: Is this change part of the ticket?

Comment (samdiano) [reply src/tests/dictionaryConcepts/container/CreateConcept.test.jsx]: I added this test to increase the coverage. There was no file related to my ticket that needed to be tested
